src/pages/AddScholarship.jsx

```diff
@@ -1,14 +1,48 @@
 import React, { useState } from 'react';
 import { Container } from '@material-ui/core';
 import ScholarshipForm from '../components/ScholarshipForm';
+import Scholarships from '../models/Scholarships';
+import SubmissionAlert from '../components/SubmissionAlert';
 
 function AddScholarship() {
   const [submissionAlert, setSubmissionAlert] = useState(null);
 
+  const scholarship = Scholarships.new({
+    name: '',
+    deadline: null,
+    description: '',
+    amount: {
+      type: null,
+      min: 0,
+      max: 0,
+    },
+    website: '',
+  });
+
   return (
     <Container maxWidth="md">
       <h1>Submit a Scholarship</h1>
-      <ScholarshipForm setSubmissionAlert={setSubmissionAlert} />
+      <ScholarshipForm
+        scholarship={scholarship}
+        submitFn={() =>
+          setSubmissionAlert(
+            <SubmissionAlert
+              id={scholarship.id}
+              name={scholarship.data.name}
+              closeFn={setSubmissionAlert(false)}
+            />
+          )
+        }
+        onSubmitError={() =>
+          setSubmissionAlert(
+            <Alert severity="error" onClose={() => setSubmissionAlert(null)}>
```

Comment:
How come this one sets the submission alert to `null` but the other sets it to `false`? Also I just noticed this property's called `onClose`, I wonder if we should have a similarly named property for the **SubmissionAlert** component instead of `closeFn`.

Reply:
oops the latter was wrong, it was suppose to set it to null as well. Although `false` also seems to work, I think `null` is more suitable.